modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/services/HealthCheckConsensus.scala

```diff
@@ -40,8 +54,15 @@ abstract class HealthCheckConsensus[F[_]: Spawn]() extends HealthCheck[F] {
           .map(Spawn[F].start)
           .sequence
           .as(ready)
-    }.flatMap { calculateOutcome }.map {
-      _.filter { case (_, (decision, _)) => decision.isInstanceOf[PositiveOutcome] }
+    }.flatMap {
+      calculateOutcome
+    }.flatTap { outcome =>
+      def positive = outcome.filter { case (_, (decision, _)) => decision.isInstanceOf[PositiveOutcome] }
+      def negative = outcome -- positive.keySet
```

Comment:
What bugs me about subtracting like this is that if we add another case and forget to handle it here it will be treated as negative outcome.

Reply:
I don't think we will extend the base but if so, I can make it the opposite way:
```
def negative = ...filter
def nonNegative = outcome -- negative.keySet
```
what do you think @TheMMaciek?
probably we don't want to make any action if the outcome is other than negative 🤔

Reply:
I guess doing it this way around is safer indeed!

Reply:
I'll do that in the next PR